src/main/java/lotto/domain/LottoRank.java

```diff
@@ -0,0 +1,39 @@
+package lotto.domain;
+
+import java.util.Arrays;
+import java.util.List;
+
+public enum LottoRank {
+
+    FIRST(6, 2_000_000_000),
+    SECOND(5, 1_500_000),
+    THIRD(4, 50_000),
+    FOURTH(3, 5_000);
+
+    private int countOfMatch;
+    private int winningMoney;
+
+    private LottoRank(int countOfMatch, int winningMoney) {
```

Comment:
enum 의 생성자는 기본으로 private 이여서 접근제어자를 생략해 줘도 좋을 것 같아요 😄 
```suggestion
    LottoRank(int countOfMatch, int winningMoney) {
```